@xen-orchestra/lite/src/locales/en.json

```diff
@@ -8,18 +8,15 @@
   "appearance": "Appearance",
   "ascending": "ascending",
   "available-properties-for-advanced-filter": "Available properties for advanced filter:",
-  "backup": "Backup",
   "back-pool-dashboard": "Go back to your Pool dashboard",
+  "backup": "Backup",
```

Comment:
I think ordering keys in precisely the right thing to do to avoid most conflicts :) 

Regarding the `appearance` key, I'm not sure to understand what you mean. The keys ordering seems to be exactly the same in [fr.json](https://raw.githubusercontent.com/vatesfr/xen-orchestra/af9437caa4914ffaed6af5fb629051aa6283d104/%40xen-orchestra/lite/src/locales/fr.json) and [en.json](https://raw.githubusercontent.com/vatesfr/xen-orchestra/af9437caa4914ffaed6af5fb629051aa6283d104/%40xen-orchestra/lite/src/locales/en.json).